src/persist-client/benches/porcelain.rs

```diff
@@ -96,9 +99,11 @@ pub fn bench_write_to_listen(
                 assert_eq!(batches.get_or_insert(b), &b);
             }
             let batches = batches.expect("didn't have at least one iter");
-            // No other known way to convert `usize` to `f64`.
-            #[allow(clippy::as_conversions)]
-            start.elapsed().div_f64(batches as f64)
+            // Justification for `unwrap`: this will
+            // succeed as long as batches <= 2^53.
+            start
+                .elapsed()
+                .div_f64(f64::try_cast_from(u64::cast_from(batches)).unwrap())
```

Comment:
> I'm a fan of these unwrap justification comments.

thoughts on making these unwrap justifications into expect messages, tho? in my mind "a justification for an unwrap" is basically the definition of expect. e.g this one could be `.expect("batches should be <= 2^53")`. I only looked at the persist files though, so feel free to ignore if this comment doesn't generalize to the rest of them

Reply:
Some of them were too long to be meaningful `expect` messages, but I suppose I could convert the ones where it's easy to describe succinctly. Will do!

Reply:
Oh yeah, great point.  I was thinking about this in the same context as some previous justification for `unsafe` comments, and totally forgot that Rust already has a mechanism for making unwrap comments explicit :-)